src/ElectricPokemon.java

```diff
@@ -2,13 +2,13 @@
 import java.util.List;
 
 public class ElectricPokemon extends Pokemon {
-    public Electric Pokemon(String name, int level, int hp, String food, String sound) {
-        super(name, level, hp, food, sound, "electric", Arrays.asList("Thunder Punch", "Electro Ball", "Thunder", "Volt Tackle"));
+    public ElectricPokemon(String name, int level, int hp, String food, String sound) {
+        super(name, level, hp, food, sound, "electric", List.of("Thunder Punch", "Electro Ball", "Thunder", "Volt Tackle"));
```

Comment:
Er zit een subtiel verschil tussen List.of en Arrays.asList. 
Over het algemeen zou ik Arrays.asList aanraden, omdat je deze gewoon kunt aanpassen (List.of is immutable) en het meer als de welbekende ArrayList gedraagd.